src/Nelmio/Alice/Fixtures/Loader.php

```diff
@@ -245,8 +250,8 @@ public function addPopulator(Populator\Methods\MethodInterface $populator)
     /**
      * parses a file at the given filename
      *
-     * @param string filename
-     * @return array data
+     * @param  string $filename
+     * @return array  data
      */
     protected function parseFile($filename)
```

Comment:
Do you think that really adds anything to the code? That feels like it can reside in the .md docs and it be sufficient.